desktop/cmp/grid/editors/InlineCheckboxEditor.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp} from '@xh/hoist/core';
+import {checkbox} from '@xh/hoist/desktop/cmp/input';
+import {useHoistInlineEditorModel} from './HoistInlineEditorModel';
```

Comment:
Replace "HoistInline" prefix with "Cell".  All will be clear from package context.